### PR TITLE
Change base.py to work with the djangobower app

### PR DESCRIPTION
**Pull request by athai**

### CODE REVIEW

**Comment by tschuy** [comment]
Please recreate this to merge into develop, not master. the master branch is reserved for releases.

**Comment by tschuy** [comment]
Also, add django-bower to the requirements/production.txt file so that it will be automatically installed at install time (that's why Travis CI is failing) :) It's best practice to pin to the version we're testing, so here 5.0.4 is what I've installed.

File: ganeti_webmgr/ganeti_web/settings/base.py [hunk 1]
```diff
@@ -175,6 +179,18 @@
...
 
+BOWER_INSTALLED_APPS = (
+    'cytoscape',
+    'jquery',
+    'jquery.cookie',
+    'jquery-form',
+    'jquery-ui',
+    'no-vnc',
+    'qTip2',
+    'raphael',
```
Comment (tschuy) [inline ganeti_webmgr/ganeti_web/settings/base.py]: Where are we using this?

**Comment by tschuy** [comment]
There's going to need to be some effort to get this working properly, but it's worth it. 

(This, along with rewriting parts of the app to use the packages brought in, will fix #58, which is a high priority for me.)

**Comment by tschuy** [comment]
For now, I'd like to reduce the list of apps to only the ones that we're storing minimized (which is a bit of a no-no):

1. jquery 1.5.2
2. jquery ui 1.8.5
3. jquery qtip 1.0.0-rc3

And be sure to keep the versions, installing the "latest" versions will most likely horribly break things. We can upgrade versions after we get this working.

**Comment by athai** [comment]
For the jquery ui, it seems that the oldest version that bower can manage goes back to 1.8.23.  What do you suggest I do here, since I can't keep it as 1.8.5?

**Comment by tschuy** [comment]
1.8.24, or the latest in the 1.8 branch, should be good. The worst that could happen is we break things, or people can now choose the date in Khmer (added in one of the 1.8 releases we don't have!)

**Comment by athai** [comment]
Okay, but now there's another problem. jquery ui 1.8.23 (which is the only option available in the 1.8 branch) requires a jquery version of at least 1.8.3.

**Comment by tschuy** [comment]
Ah. I was hoping to avoid upgrading jQuery.

Grab the latest version of 1.8, I guess. I'll have to do an annoying amount of testing.

**Comment by tschuy** [comment]
Closing in favor of #61

**Pull request closed without merging**